browser/components/pages/NotFoundPage.js

```diff
@@ -1,12 +1,47 @@
 import React, { Component } from 'react'
+import sadCat1 from '../../images/sadcat1.gif'
+import sadCat2 from '../../images/sadcat2.gif'
+import sadCat3 from '../../images/sadcat3.gif'
+import sadCat4 from '../../images/sadcat4.gif'
 import Link from '../atoms/Link'
+import './NotFoundPage.sass'
 
 export default class NotFoundPage extends Component {
+
+  whichSadCat() {
+    let num = Math.floor( Math.random() * 4 )
+    if ( num === 0 ) {
+      return sadCat1
+    }
+    else if ( num === 1 ) {
+      return sadCat2
+    }
+    else if ( num === 2 ) {
+      return sadCat3
+    }
+    else {
+      return sadCat4
+    }
+  }
```

Comment:
This shows me you need to work on your algorithms. Please take a shot at significantly simplifying this method.